developer/src/kmc-ldml/src/compiler/messages.ts

```diff
@@ -149,5 +149,9 @@ export class CompilerMessages {
   m(this.ERROR_DisplayNeedsToOrId, `display ${CompilerMessages.toOrId(o)} needs to= or id=, but not both`);
   static ERROR_DisplayNeedsToOrId = SevError | 0x0022;
 
+  static Hint_UnsupportedCustomForm = (o:{id: string}) =>
+  m(this.HINT_UnsupportedCustomForm, `Custom <form id="${o.id}"> element is not supported.`);
+  static HINT_UnsupportedCustomForm = SevHint | 0x0023;
```

Comment:
Should this be a hint or a warning? Does the custom form cause breaking issues down the line?

Reply:
maybe a warning is better than a hint. Basically, it's not processed by kmc.